pkgs/development/python-modules/gevent-eventemitter/default.nix

```
@@ -0,0 +1,36 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  coverage,
+  gevent,
+  pytestCheckHook,
+}:
+
+buildPythonPackage rec {
+  pname = "gevent-eventemitter";
+  version = "2.1";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "rossengeorgiev";
+    repo = "gevent-eventemitter";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-aW4OsQi3N5yAMdbTd8rxbb2qYMfFJBR4WQFIXvxpiMw=";
+  };
+
+  build-system = [ setuptools ];
+  dependencies = [ gevent ];
+  nativeCheckInputs = [
+    coverage
+    pytestCheckHook
```

```suggestion
    pytestCheckHook
    pytest-cov-stub
    coveralls
```
From dev_requirements.txt